It seems that the test has to be improved.
```
>>> Sum(sin(n)/log(log(n)), (n, 22, oo)).is_convergent()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/concrete/summations.py", line 442, in is_convergent
    if abs(lim_ratio) > 1:
  File "sympy/core/relational.py", line 229, in __nonzero__
    raise TypeError("cannot determine truth value of Relational")
TypeError: cannot determine truth value of Relational
```
In this case, the limit cannot be computed and `lim_ratio` is not a number. That should also be tested.
(And `S.Zero` could be 0 as it will be sympified anyway.)
There could also be a unit test for this example.

--- a/sympy/concrete/summations.py
+++ b/sympy/concrete/summations.py
@@ -464,8 +464,7 @@ class Sum(AddWithLimits, ExprWithIntLimits):
         # (1/n) comparison
         try:
             lim_comp = limit(sym*sequence_term, sym, S.Infinity)
-            if lim_comp.is_number:
-                if lim_comp > 0:
+            if lim_comp.is_number and lim_comp > 0:
                     return S.false
         except NotImplementedError:
             pass

--- a/sympy/concrete/summations.py
+++ b/sympy/concrete/summations.py
@@ -439,10 +439,11 @@ class Sum(AddWithLimits, ExprWithIntLimits):
         next_sequence_term = sequence_term.xreplace({sym: sym + 1})
         ratio = combsimp(powsimp(next_sequence_term/sequence_term))
         lim_ratio = limit(ratio, sym, upper_limit)
-        if abs(lim_ratio) > 1:
-            return S.false
-        if abs(lim_ratio) < 1:
-            return S.true
+        if lim_ratio.is_number:
+            if abs(lim_ratio) > 1:
+                return S.false
+            if abs(lim_ratio) < 1:
+                return S.true
 
         ### --------- p-series test (1/n**p) ---------- ###
         p1_series_test = order.expr.match(sym**p)
@@ -463,8 +464,9 @@ class Sum(AddWithLimits, ExprWithIntLimits):
         # (1/n) comparison
         try:
             lim_comp = limit(sym*sequence_term, sym, S.Infinity)
-            if lim_comp > S.Zero:
-                return S.false
+            if lim_comp.is_number:
+                if lim_comp > 0:
+                    return S.false
         except NotImplementedError:
             pass
 

--- a/sympy/concrete/tests/test_sums_products.py
+++ b/sympy/concrete/tests/test_sums_products.py
@@ -1019,3 +1019,6 @@ def test_issue_14112():
 
 def test_issue_14111():
     assert Sum(1/log(log(n)), (n, 22, oo)).is_convergent() is S.false
+
+def test_issue_14484():
+    raises(NotImplementedError, lambda: Sum(sin(n)/log(log(n)), (n, 22, oo)).is_convergent())

